.github/workflows/build-yocto.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       run: |
         cd Build
         ./run.sh -c /home/github/rzg2l-cache
-        cp output/images/smarc-rzv2l/*.wic.bz2 /home/github
+        cp output/images/smarc-rzg2l/*.wic.bz2 /home/github
```

Comment:
Does this have to be hardcoded? Can we have a variable in the path that gets set to either g2l or v2l?